package/boot/uboot-sunxi/uEnv-h6.txt

```diff
@@ -0,0 +1,7 @@
+setenv mmc_rootpart 2
+part uuid mmc ${mmc_bootdev}:${mmc_rootpart} uuid
+setenv loadkernel fatload mmc \$mmc_bootdev \$kernel_addr_r uImage
+setenv loaddtb fatload mmc \$mmc_bootdev \$fdt_addr_r dtb
+setenv bootargs console=ttyS0,115200 earlyprintk root=PARTUUID=${uuid} rootwait
```

Comment:
The only differences to A64 is only the removed ```earlycon=uart,mmio32,0x01c28000``` correct?

Reply:
That is correct.